<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UiIPSreach</class>
 <widget class="QDialog" name="UiIPSreach">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>547</width>
    <height>525</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="labServerPort_4">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>390</y>
     <width>81</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>发送信息</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="ToConnect">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>30</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>连接</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Disconnect">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>90</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>断开</string>
   </property>
  </widget>
  <widget class="QLabel" name="labServerPort_2">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>120</y>
     <width>81</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>远程端口</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QListWidgetMouse" name="Mouselist">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>181</width>
     <height>331</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labServerPort_3">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>310</y>
     <width>81</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>接受信息</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="IP">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>24</y>
     <width>144</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="FindIP">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>430</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>搜索</string>
   </property>
  </widget>
  <widget class="QLabel" name="labServerIP_2">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>50</y>
     <width>71</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>远程地址</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labServerPort_5">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>20</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>IP地址</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="ClientSend">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>430</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Port">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>90</y>
     <width>144</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="MsgMain">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>150</y>
     <width>271</width>
     <height>151</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="MainSend">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>350</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>480</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>关闭</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QListWidgetMouse</class>
   <extends>QListWidget</extends>
   <header location="global">qlistwidgetmouse.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
